src/main/java/svc/controllers/SMSController.java

```diff
@@ -0,0 +1,160 @@
+package svc.controllers;
+
+import java.io.IOException;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.*;
+import com.twilio.twiml.Body;
+import com.twilio.twiml.Message;
+import com.twilio.twiml.MessagingResponse;
+import com.twilio.twiml.TwiMLException;
+
+import javax.inject.Inject;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import svc.dto.CitationSearchCriteria;
+import svc.logging.LogSystem;
+import svc.managers.*;
+import svc.managers.SMSManager.SMS_STAGE;
+import svc.models.*;
+
+@RestController
+@EnableAutoConfiguration
+@RequestMapping("/sms")
+public class SMSController {
+	@Inject
+	SMSManager smsManager;
+	@Inject
+	CitationManager citationManager;
+	
+	String dob,license;
+	SimpleDateFormat dateFormat;
+	CitationSearchCriteria criteria;
+	List<Citation> citations;
+	
+	public static final String ACCOUNT_SID = "AC5d1ac1d852d5ee86bdb7e92bedc524ab";
+	public static final String AUTH_TOKEN = "a636e8943be534b338147adfe6a8ba01";
+	
+	@ResponseBody
+	@RequestMapping(method = RequestMethod.POST)
+	void GetMessage(@ModelAttribute TwimlMessageRequest twimlMessageRequest, HttpServletResponse response, HttpSession session) throws IOException{
+		session.setMaxInactiveInterval(30*60); //set session timeout to 30 minutes
```

Comment:
There is a TON of business logic in here. Can you move most of this into a new SMSManager like we have managers for other things? Controllers really need to be kept super-simple or they become too inflexible and the logic isn't reusable. Business logic should be delegated off to composable services. Just at a glance, I don't think this method should be more than 3-5 lines long with everything else in a service/manager.